runtime/doc/autocmd.txt

```diff
@@ -987,8 +987,8 @@ TermClose			When a |terminal| job ends.
 				Sets these |v:event| keys:
 				    status
 							*TermResponse*
-TermResponse			When Nvim receives a OSC response from the
-				terminal.  Sets |v:termresponse|. When used
+TermResponse			When Nvim receives an OSC or DCS response from
+				the terminal.  Sets |v:termresponse|. When used
```

Comment:
Is the "pattern" storing anything useful? If not should it say "osc" or "dsc" as appropriate, so that handlers can easily ignore responses they aren't interested in?

For the `TermOSC` / `TermRequest` event in https://github.com/neovim/neovim/pull/22159 I was thinking we would put osc/dsc/etc in the event pattern.

Reply:
See my comment [here](https://github.com/neovim/neovim/pull/26061#discussion_r1395753206).